lib/internal/modules/esm/resolve.js

```diff
@@ -1078,6 +1070,7 @@ module.exports = {
   packageExportsResolve,
   packageImportsResolve,
   throwIfInvalidParentURL,
+  legacyMainResolve,
```

Comment:
you are exporting it just for benchmark?

Reply:
Yes